LICENSE

```diff
@@ -1,3 +1,6 @@
+pygatt:
```

Comment:
Have you talked to Peplin about how authorship, licensing, attribution, etc. should work for this?  I'm not 100% on all of that.